mmv1/third_party/terraform/website/docs/r/dataproc_workflow_template.html.markdown

```diff
@@ -782,143 +782,160 @@ cluster_config {
-    
+  The name of the on-cluster Kerberos realm. If not specified, the uppercased domain of hostnames will be the realm.
+
 * `root_principal_password` -
   (Optional)
-  Optional. The Cloud Storage URI of a KMS encrypted file containing the root principal password.
-    
+  The Cloud Storage URI of a KMS encrypted file containing the root principal password.
+
 * `tgt_lifetime_hours` -
   (Optional)
-  Optional. The lifetime of the ticket granting ticket, in hours. If not specified, or user specifies 0, then default value 10 will be used.
-    
+  The lifetime of the ticket granting ticket, in hours. If not specified, or user specifies 0, then default value 10 will be used.
+
 * `truststore` -
   (Optional)
-  Optional. The Cloud Storage URI of the truststore file used for SSL encryption. If not provided, Dataproc will provide a self-signed certificate.
-    
+  The Cloud Storage URI of the truststore file used for SSL encryption. If not provided, Dataproc will provide a self-signed certificate.
+
 * `truststore_password` -
   (Optional)
-  Optional. The Cloud Storage URI of a KMS encrypted file containing the password to the user provided truststore. For the self-signed certificate, this password is generated by Dataproc.
-    
+  The Cloud Storage URI of a KMS encrypted file containing the password to the user provided truststore. For the self-signed certificate, this password is generated by Dataproc.
+
 The `software_config` block supports:
-    
+
 * `image_version` -
   (Optional)
-  Optional. The version of software inside the cluster. It must be one of the supported (https://cloud.google.com/dataproc/docs/concepts/versioning/dataproc-versions#other_versions). If unspecified, it defaults to the latest Debian version.
-    
+  The version of software inside the cluster. It must be one of the supported (https://cloud.google.com/dataproc/docs/concepts/versioning/dataproc-versions#other_versions). If unspecified, it defaults to the latest Debian version.
+
+* `optional_components` -
+  (Optional)
+  The set of components to activate on the cluster.
+
 * `properties` -
   (Optional)
-  Optional. The properties to set on daemon config files. Property keys are specified in `prefix:property` format, for example `core:hadoop.tmp.dir`. The following are supported prefixes and their mappings: * capacity-scheduler: `capacity-scheduler.xml` * core: `core-site.xml` * distcp: `distcp-default.xml` * hdfs: `hdfs-site.xml` * hive: `hive-site.xml` * mapred: `mapred-site.xml` * pig: `pig.properties` * spark: `spark-defaults.conf` * yarn: `yarn-site.xml` For more information, see (https://cloud.google.com/dataproc/docs/concepts/cluster-properties).
-    
+  The properties to set on daemon config files.
+
+  Property keys are specified in prefix:property format, for example core:hadoop.tmp.dir. The following are supported prefixes and their mappings:
+
+  capacity-scheduler: capacity-scheduler.xml
```

Comment:
I think you'll still need the bullet prefixes to indicate an unordered list.

You can test whether it will display properly with the doc preview tool: https://registry.terraform.io/tools/doc-preview